### PR TITLE
Add ZargCon Information Page

### PR DESCRIPTION
**Pull request by smunilla**

### CODE REVIEW

**Comment by bmcelvee** [comment]
Adding that the snacks/drinks on premise are not available for the event and LGTM

File: zargcon.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+In 2019, we are being hosted by Red Hat Tower Annex.
+
+* Because this is the place of employment for many of our members, we ask that all attendees take extra care to respect the venue
+* Drinks and snacks provided by Red Hat will not be available to our attendees
+
+## What should I bring?
+
+* Yourself
+* Any games you want to play
```
Comment (awood) [inline zargcon.md]:
```suggestion
* Any games you want to play.  (Please label your games as belonging to you somehow.  E.g. a sticker on the inside box cover, a business card in the game components, etc.  Also make a list of the games you are bringing so you can double check that you don't forget to bring anything home.) 
```

File: zargcon.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## FAQ
+
+### Can I invite other people?
+
+As long as you're cool with them and you think they won't be jerks, sure! They'll still have to pay however. We will also need a list names for any of your guests.
+
+### Is there a registration cut off?
+
+Yes. Monday Dec 2. Have your registration fee paid and provide your guest list by then.
```
Comment (awood) [inline zargcon.md]: We also need to tell people how exactly they can register.

Comment (smunilla) [reply zargcon.md]: Edited.

File: zargcon.md (hunk 1 above)
Comment (smunilla) [reply zargcon.md]: I added this as an event rule.

**Pull request merged**